gocov/test.go

```diff
@@ -86,12 +90,24 @@ func runTests(args []string) error {
 		// the JSON coverage output.
 		cmd.Stdout = os.Stderr
 		cmd.Stderr = os.Stderr
-		err := cmd.Run()
+
+		commandChan <- cmd
+	}
+
+	for i := 0; i < runtime.NumCPU(); i++ {
```

Comment:
please add "-p" to the common build flags in internal/testflag, and have that default to NumCPU()